docs/stencil-docs/developing-further/stored-credit-card-management.mdx

```diff
@@ -84,6 +84,110 @@ curl -O https://raw.githubusercontent.com/leeBigCommerce/cornerstone/54f5681a6a1
 ## Update `en.json`
 At minimum, a translation for the `en` needs to be added to `lang/en.json: [Pull Request #1603](https://github.com/bigcommerce/cornerstone/pull/1603/files?file-filters%5B%5D=.html&file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-b0d4c1fc9d8d2a5a213b27a72cf6c9fe). Add translations for other locales as needed.
 
+## Manage stored account payments
+
+After adding a stored payment method, shoppers can use the Storefront account payments microapp to manage their stored payment instruments. The Storefront account payments provide a payment method form with billing address fields. Currently, the microapp works only with Ayden.
+
+### Microapp installation instructions
+
+To install the Storefront account microapp to your theme, inject the `account_payments` variable into your template using `add-payment-method.html`.
+
+```js showLineNumbers copy
+{{#if account_payments}}
+    {{{ account_payments }}}
+    <script>
+        window.BigCommerce = window.BigCommerce || {};
+    </script>
+{{/if}}
+```
+`account_payments` will provide the `<div>` element where you can render Storefront account payments and attach the renderAccountPayments() method to the window.BigCommerce object that displays the Storefront account payments app if the selected payment provider is already implemented.
+
+A [Cornerstone example](https://github.com/bigcommerce/cornerstone/blob/master/templates/pages/account/add-payment-method.html#L45) shows how to load a script to the page. In this example, we have a condition: if `account_payments` is available - it renders; if not - the fallback payment method form renders.
+
+In the subsequent releases, we'll remove the fallback payment method form because the Storefront account payments microapp will support all payment methods.

```

Comment:
rephrase with something like `Our roadmap includes plans for the Storefront accounts payment microapp to support all payment methods, ...( then the fallback payment method form will go away)`